### DISCUSSION TITLE
Добовление табличек

### DISCUSSION THREAD

**Discussion by SIANIE19HW** [Sokol-Games-GmbH/PixelCombats.PublicApi#211] [Ideas]
Данную идею я хочу использовать чтобы не писать текст блоками так как это очень долго и некрасиво. Идея табличек заключается в том что писать на них текст той или иной вам величиной какую поставите вы. Данная идея поможет в создании карт.

**Comment by kkohno** [comment]
идея актуальная - какраз ищем как в АПИ игроков лучше информировать в 3Д мире, что-то типо табличек... опишите подробнее

**Comment by SIANIE19HW** [comment]
Таблички - это как передача информации с помощью их, например в ТДМ. Если карта с табличкой на которой написанно например где спавн синих или наоборот или какой-то объект
